src/directives/for-of/README.md

```diff
@@ -77,21 +77,30 @@ When the contents of the iterator changes, `igxForOf` makes the corresponding ch
 
 ### Accessors
 
-List of public accessors that the developers may use to get information from the `igxFor`:
-| Name | Type | Description |
-| :--- |:--- | :--- |
-| id | string | Unique identifier of the directive |
-| state | IgxForState | The current state of the directive it contains `startIndex` and `chunkSize` |
-| totalItemCount | number | The total count of the virtual data items, when using remote service |
+List of public accessors that the developers may use to get information from the `igxForOf`:
+| Name           | Type        | Description                                                                 |
+| :------------- |:----------- | :-------------------------------------------------------------------------- |
+| id             | string      | Unique identifier of the directive                                          |
+| state          | IgxForState | The current state of the directive it contains `startIndex` and `chunkSize` |
```

Comment:
Add a comma - 'The current state of the directive, it contains..."
and what are `startIndex` and `chunkSize`? purpose?